app/views/layouts/_sidebar.html.erb

```diff
@@ -57,9 +57,16 @@
           <%= lucide_icon("megaphone", class: "w-5 h-5 text-gray-500 shrink-0") %>
           <span class="text-gray-900 text-sm">Feedback</span>
         <% end %>
-        <%= link_to "https://link.maybe.co/discord", class: "flex gap-2 items-center hover:bg-gray-50 rounded-lg px-3 py-2" do %>
-          <%= lucide_icon("message-square-more", class: "w-5 h-5 text-gray-500 shrink-0") %>
-          <span class="text-gray-900 text-sm">Contact</span>
+        <% if ENV["SELF_HOSTED"] == "true" %>
```

Comment:
```suggestion
        <% if self_hosted? %>
```

In all view-related code, I've consolidated this to `self_hosted?` helper in `SelfHostable` just to consolidate changes to one centralized spot 

https://github.com/maybe-finance/maybe/blob/main/app/controllers/concerns/self_hostable.rb